docs/containers-storage.conf.5.md

```diff
@@ -168,11 +168,38 @@ The `storage.options.overlay` table supports the following options:
   ignore_chown_errors can be set to allow a non privileged user running with a  single UID within a user namespace to run containers. The user can pull and use any image even those with multiple uids.  Note multiple UIDs will be squashed down to the default uid in the container.  These images will have no separation between the users in the container. (default: false)
 
 **force_mask** = "0000|shared|private"
-  force_mask sets an octal permission mask that is used for the new files and directories.  The values "native", "shared", "private" are also accepted.  (default: "native")
-  "private": it is equivalent to 0700.  The owner has rwx access to the files.
-  "shared": it is equivalent to 0755.  The owner has rwx access to the files and everyone else can read, access and execute them.
-  Note: The force_mask Flag is an experimental feature, it could change in the future.
-  When "force_mask" is used then the original permission mask is stored in the "user.containers.override_stat" xattr.
+  ForceMask specifies the permissions mask that is used for new files and
+directories.
+The values "native", "shared", "private" are accepted.  (default: "native").
```

Comment:
that was a mistake in my version.  There is no "native" anymore.  The default is the empty string.